html_utils/scraping.py

```diff
@@ -0,0 +1,11 @@
+import requests
+from bs4 import BeautifulSoup
+
+HTML_PARSER = "html.parser"
+
+def do_get_request(url: str) -> requests.Response:
```

Comment:
missing docstrings

Reply:
why do you need to encapsulate this in a different call?

Reply:
Because we can decouple the requests module so we can change the way to do a get request, validations, decorators, etc.